website/docs/components/side-nav/partials/specifications/anatomy.md

```diff
@@ -1,7 +1,7 @@
 ## Anatomy
 
 <!-- image then table -->
-![Anatomy of sideNav](/assets/components/side-nav/side-nav-anatomy.png)
+![Anatomy of sideNav](/assets/components/side-nav/side-nav-anatomy.png =580x*)
```

Comment:
@cveigt I fixed the rendered size (width) of the image. (It's a 2x/retina image so needs to display at half the actual pixel size)